.sling-module.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
@stefanseifert Sure, temporarily disabled Windows builds because one of the test cases is failing. Unfortunately, addressing the issue is proving to be a bit challenging with MacBook. However, I'm exploring the possibility of using Windows Docker instances to see if I can tackle the problem from that angle.

Do you have or anyone has suggestions or alternative approaches might help here.

Reply:
i was able to build it locally on my windows 11 machine without problems - did not had time to dig deeper into the jenkins logs yet